resources/views/components/checkboxes.blade.php

```diff
@@ -1,11 +1,11 @@
-@foreach($options as $value => $atts)
+@foreach($options as $option)
 <div class="field">
-    @php($id = $name . '-' . Str::slug($value))
-    @php($hint = isset($atts['hint']) ? $id . '-hint' : '')
-    <input {!! $attributes !!} type="checkbox" name="{{ $name }}[]" id="{{ $id }}" value="{{ $value }}" {!! $describedBy($hint) ? 'aria-describedby="' . $describedBy($hint) . '"' : '' !!} @checked(in_array($value, $checked, true)) {!! $invalid ? 'aria-invalid="true"' : '' !!} />
-    <x-hearth-label for="{{ $id }}">{{ $atts['label'] }}</x-hearth-label>
-    @if(isset($atts['hint']))
-    <x-hearth-hint for="{{ $id }}">{{ $atts['hint'] }}</x-hearth-hint>
+    @php($id = $name . '-' . Str::slug($option['value']))
+    @php($hint = isset($option['hint']) ? $id . '-hint' : '')
+    <input {!! $attributes !!} type="checkbox" name="{{ $name }}[]" id="{{ $id }}" value="{{ $option['value'] }}" {!! $describedBy($hint) ? 'aria-describedby="' . $describedBy($hint) . '"' : '' !!} @checked(in_array($option['value'], $checked)) {!! $invalid ? 'aria-invalid="true"' : '' !!} />
+    <x-hearth-label for="{{ $id }}">{{ $option['label'] }}</x-hearth-label>
+    @if(isset($option['hint']) && !empty($option['hint']))
```

Comment:
would just calling `@if($option['hint'])` work in this case?

Reply:
Yes, it appears to work. I had an issue before where empty paragraphs were being rendered when the hint was an empty string (this can happen with [spatie/laravel-options](https://github.com/spatie/laravel-options)) but I think it's safe to remove the extra conditional.

Reply:
Actually, I take that back. It is necessary to have both checks because sometimes there's no `hint` key. In that instance, `@if($options['hint'])` throws an `Undefined array key "hint"` error. But we don't want to render the hint if the hint key is present but the value is an empty string, so we need to check both that: 1) the array key is present; 2) it has a value.